theatre/studio/src/uiComponents/form/BasicNumberInput.tsx

```diff
@@ -275,14 +258,12 @@ const BasicNumberInput: React.FC<{
     return {
       inputChange,
       onBlur,
-      transitionToDraggingMode,
       onInputKeyDown,
       onClick,
-      onFocus,
       onDragEnd,
       onDrag,
     }
-  }, [])
+  }, []) // TODO: fix the missing dependency warning
 
   let value =
     stateRef.current.mode !== 'editingViaKeyboard'
```

Comment:
Does the user need the ability to edit more that 3 decimal places? 

Now that the user can drag whilst already focused (ie without clicking away first) it will display the extra decimal places for a split second (whilst `mode === 'editingViaKeyboard'). It's relatively minor, but a simple fix would be to only show a max of 3 decimals

https://user-images.githubusercontent.com/1253802/154534381-0900bcab-85d0-419a-b311-566aa8a1d995.mov